src/rohit/Commonelement.java

```diff
@@ -0,0 +1,32 @@
+package rohit;
+
+/*Program 1: WAP to Find Common Element Between Two Arrays
+
+input:
+
+int array1 [] = [1, 2, 5, 5, 8, 9, 7, 10]
+int array2 [] = [1, 0, 6, 15, 6, 4, 7, 0]
+
+output: 1 7*/
+public class Commonelement {
+
+	void arraycommonelement(int[] array1, int[] array2) {
+		System.out.println("Common elements are: ");
+		for (int i = 0; i < array1.length; i++) {
+			for (int j = 0; j < array2.length; j++) {
+				if (array1[i] == (array2[j])) {
+
+					System.out.println((array1[i]));
+				}
```

Comment:
You just need to put break, once you match the element value in second array

Reply:
ok will take care of that from next time took 40 mins time to complete it, thanks for your comments.